docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -41,7 +41,33 @@ git clone https://github.com/YOUR_NAME/autoware.git
 
 If you set up multiple types of vehicles, adding a suffix like "autoware.vehicle_A" or "autoware.vehicle_B" is recommended.
 
-## 4. Create the description packages of your vehicle
+## 4. Create your Autoware Launch repository
```

Comment:
The repository is referred to as `Autoware Launch` a few times (see lines 57 and 200), but the README file says `autoware_launch`. Which name is correct?